services/provider/server/server.go

```diff
@@ -553,13 +557,22 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 	var storageClassName string
 	storageClassData := map[string]string{}
 
+	storageClassRequestHash := getStorageClassRequestHash(req.StorageConsumerUUID, req.StorageClassClaimName)
+	nodeSecretName := scrCephClientCsiSecretName("node", storageClassRequestHash)
+	provisionerSecretName := scrCephClientCsiSecretName("provisioner", storageClassRequestHash)
+
+	storageClassData["csi.storage.k8s.io/provisioner-secret-name"] = provisionerSecretName
+	storageClassData["csi.storage.k8s.io/node-stage-secret-name"] = nodeSecretName
+	storageClassData["csi.storage.k8s.io/controller-expand-secret-name"] = provisionerSecretName
```

Comment:
I was under the impression we are not moving any more code outside the switch case and that we are going to refactor the code inside the switch case into its own function in the next commit. I agreed to keep the  `nodeSecretName` and `provisionerSecretName` based on that. 

This change is the exact opposite of that agreement. What changed?

Reply:
Since I'm the one that's doing the follow-on PR, I'm organizing it in a way that makes sense for me to then move it elsewhere.

Reply:
ok